src/System Application/App/ControlAddIns/src/WebPageViewer/js/WebPageViewer.js

```diff
@@ -0,0 +1,175 @@
+/*! Copyright (C) Microsoft Corporation. All rights reserved. */
+
+var iframe = null;
+var subscribeToSrcLoad = function () { };
+var maxSize = 300;
+var defaultSize = '100%';
+var iframeHeight = defaultSize, iframeWidth = defaultSize;
+var pageTabSize = 20; // is 20 height of page tabs?
```

Comment:
Looks like a TODO left here

Reply:
Not created by Stig and seems to have been there forever. I'd suggest to just remove or leave the comment.

Reply:
Removed